documentation/asciidoc/computers/remote-access/secure-shell-passwordless.adoc

```diff
@@ -122,3 +122,8 @@ Run the following command to store it in your keychain:
 ----
 ssh-add -K ~/.ssh/id_rsa
 ----
+
+[NOTE]
+====
+From macOS Monterey onwards the `-K` has been deprecated and have been replaced by the `--apple-use-keychain` flag.
```

Comment:
"have been replaced" -> "has been replaced" ?

Reply:
Reworded in b089e9e